### ISSUE TITLE
sbt.bat doesn't work on windows with java9

### ISSUE THREAD

**Issue by js75gh46s** [sbt/sbt-launcher-package#190]
The file sbt.bat needs to use delayed expansion in copyrt function.  The environment variables rtexport, java9_ext, and java9_rt are set inside the if statement, the uses of these variables need the delayed expansion ( !java9_rt! instead of %java9_rt% )

Below is a version that works.

```
:copyrt
if /I "%JAVA_VERSION%" GEQ "9" (
  set rtexport="%SBT_HOME%java9-rt-export.jar"

  "%_JAVACMD%" %_JAVA_OPTS% %SBT_OPTS% -jar "!rtexport!" --rt-ext-dir > "%TEMP%.\rtext.txt"
  set /p java9_ext= < "%TEMP%.\rtext.txt"
  set java9_rt=!java9_ext!\rt.jar

  if not exist "!java9_rt!" (
    echo Copying runtime jar.
    mkdir "!java9_ext!"
    "%_JAVACMD%" %_JAVA_OPTS% %SBT_OPTS% -jar "!rtexport!" "!java9_rt!"
  )
  set _JAVA_OPTS=!_JAVA_OPTS! -Dscala.ext.dirs="!java9_ext!"
)
exit /B 0
```

**Comment by dsquier** [comment]
I encountered an additional problem with the 1.1.0 Windows MSI installer when I installed to the default directory (i.e., `C:\Program Files (x86)`) which contains spaces.

I was able to fix this by uninstalling and re-installing to a directory without spaces (i.e., `C:\sbt`).

**Comment by eed3si9n** [comment]
The delayed expansion change was done in https://github.com/sbt/sbt-launcher-package/pull/201.
There was still problem with 1.1.0 so I just released 1.1.0.1
